common/src/main/java/io/netty/util/internal/ClassInitializerUtil.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util.internal;
+
+/**
+ * Utility which ensures that classes are loaded by the {@link ClassLoader}.
+ */
+public final class ClassInitializerUtil {
+
+    private ClassInitializerUtil() { }
+
+    /**
+     * Preload the given classes and so ensure the {@link ClassLoader} has these loaded after this method call.
+     *
+     * @param classLoader   the {@link ClassLoader}
+     * @param classNames    the classes to load.
+     */
+    public static void tryLoadClasses(ClassLoader classLoader, String... classNames) {
```

Comment:
This may not work for shading?

Reply:
@dapengzhang0 I think I could make this private as how we use it (with `Class`) it works with shading :)

Reply:
It looks unused. Delete it? The Class-based version wouldn't depend on the shading tool.